engine/src/io/data_provider/UriDataProvider.cpp

```diff
@@ -117,7 +114,7 @@ data_handle uri_data_provider::get_next() {
 
 			if(fs_manager && fs_manager->exists(target_uri)) {
 				fileStatus = BlazingContext::getInstance()->getFileSystemManager()->getFileStatus(target_uri);
-			} else {
+			} else if (!ignore_missing_paths){
```

Comment:
why would we not want to tell the user we couldnt find a file they expected?

Reply:
See my comment above